src/scripts/updateTasks.js

```diff
@@ -0,0 +1,52 @@
+import { getLocal, updateLocal } from './localStorage.js';
+import newIndex from './updateIndex.js';
+
+function changeStatus(done, tasks, task) {
+  const local = getLocal();
+  if (local) {
+    tasks = local;
+    tasks = newIndex(tasks);
+  }
+  const objIndex = tasks.findIndex((obj) => obj.index === task.index);
+  if (done.checked) {
+    tasks[objIndex].completed = true;
+    updateLocal(tasks);
+  } else {
+    tasks[objIndex].completed = false;
+    updateLocal(tasks);
+  }
+}
+
+function editTask(editBtn, description, tasks, task) {
```

Comment:
- [ ] [OPTIONAL]   It's better to void a long number of arguments. Ideally, a function should have two or fewer arguments specified. The fewer the arguments, the easier is to test the function.